packages/components/src/molecules/Rating/Rating.tsx

```diff
@@ -16,9 +16,9 @@ export interface RatingProps
    */
   value: number
   /**
-   * A React component that will be rendered as an icon.
+   * A React component that will be rendered as an icon rating.
    */
```

Comment:
I think "be rendered as an icon rating" still doesn't tell the user much about how this prop will be used. How about something like: "Icon to represent a rating score unit (ex: a `<Star />` component)"

Reply:
much better :)